docs/sources/helm-charts/mimir-distributed/migration-guides/migrate-from-cortex.md

```diff
@@ -252,4 +253,13 @@ You can migrate to the Grafana Mimir Helm chart (`grafana/mimir-distributed` v3.
    helm upgrade <RELEASE> grafana/mimir-distributed [-n <NAMESPACE>]
    ```
 
-To verify that the cluster is operating correctly, use the [monitoring mixin dashboards](/docs/mimir/{{< param "mimir_docs_version" >}}/manage/monitor-grafana-mimir/dashboards/).
+To verify that the cluster is operating correctly, use the [monitoring mixin dashboards].
+
+{{% docs/reference %}}
+[Migrate from Cortex]: "/ -> /docs/mimir/<MIMIR DOCS VERSION>/migrate/migrate-from-cortex"
+[mimirtool_rules]: "/ -> /docs/mimir/<MIMIR DOCS VERSION>/operators-guide/tools/mimirtool#rules"
+[`mimirtool config convert`]: "/ -> /docs/mimir/<MIMIR DOCS VERSION>/operators-guide/tools/mimirtool#convert"
+[convert]: "/ -> /docs/mimir/<MIMIR DOCS VERSION>/operators-guide/tools/mimirtool#convert"
+[Additional resources metrics]: "/ -> /docs/mimir/<MIMIR DOCS VERSION>/operators-guide/monitor-grafana-mimir/requirements#additional-resources-metrics"
+[monitoring mixin dashboards]: "/ -> /docs/mimir/<MIMIR DOCS VERSION>/operators-guide/monitor-grafana-mimir/dashboards"
```

Comment:
```suggestion
[mimirtool_rules]: "/ -> /docs/mimir/<MIMIR DOCS VERSION>/manage/tools/mimirtool#rules"
[`mimirtool config convert`]: "/ -> /docs/mimir/<MIMIR DOCS VERSION>/manage/tools/mimirtool#convert"
[convert]: "/ -> /docs/mimir/<MIMIR DOCS VERSION>/manage/tools/mimirtool#convert"
[Additional resources metrics]: "/ -> /docs/mimir/<MIMIR DOCS VERSION>/manage/monitor-grafana-mimir/requirements#additional-resources-metrics"
[monitoring mixin dashboards]: "/ -> /docs/mimir/<MIMIR DOCS VERSION>/manage/monitor-grafana-mimir/dashboards"
```

Reply:
This suggestion is not for the current release of the Mimir documentation. There is no "Manage" section in the `v2.9.x` documentation (https://grafana.com/docs/mimir/v2.9.x/).

Reply:
Ah, ok. I think we should add a milestone to this PR.